test/OWNERS

```diff
@@ -44,31 +33,34 @@ approvers:
   - MrHohn
   - deads2k
   - enisoc
-  - enj # for test/integration/etcd/etcd_storage_path_test.go
-  - eparis
-  - erictune
-  - foxish # for test/e2e/network-partition.go
```

Comment:
When moving someone to emeritus we should probably consider preserving the comments like this.

Reply:
@nikhita @spiffxp

Reply:
Good point! cc @mrbobbytables

Reply:
💯 can update guidelines on preserving or adding comments like that.